_specs_/websocket/schemas/common/events/auth.yml

```
@@ -0,0 +1,11 @@
+allOf:
+  - $ref: './event.yml'
```

```suggestion
  - $ref: '../../index.yml#/Event'
```

Some of the other events reference the object via the index.